ONSdigital/dp-frontend-renderer
# Remove metadata footer

### What

Remove metadata footer from dataset landing page and dataset filter/download journeys.

### How to review

1) Checkout out the dataset controller and fitler dataset controller to `feature/remove-metadata-footer`
2) Check the metadata footer no longers shows and the application builds fine.

### Who can review

Anyone but me.
